intersectionLinkedLists.js

```diff
@@ -0,0 +1,37 @@
+// This file contains solution to the following problem:
+// https://leetcode.com/problems/intersection-of-two-linked-lists/
+
+const getIntersectionNode = (headA, headB) => {
+  const countDepth = (head) => {
+    let depth = 0;
+    while (head) {
+      head = head.next;
+      depth += 1;
+    }
+    return depth;
+  }
+  const depthA = countDepth(headA);
+  const depthB = countDepth(headB);
+
+  let skippedHead = headA;
+  let otherHead = headB;
+  let skipCnt = depthA - depthB;
+
+  if (depthB > depthA) {
+    skipCnt = depthB - depthA;
+    skippedHead = headB;
+    otherHead = headA;
+  }
```

Comment:
No need to have this newline splitting sections of code related to each other, lets remove it (it's more readable when related code is groupped together). Although in general it's hard to there are no strict rules for this - you'll just have to guess.